source/extensions/extensions_build_config.bzl

```diff
@@ -7,6 +7,12 @@ EXTENSIONS = {
     "envoy.access_loggers.file":                        "//source/extensions/access_loggers/file:config",
     "envoy.access_loggers.http_grpc":                   "//source/extensions/access_loggers/http_grpc:config",
 
+    #
+    # gRPC Credentials
+    #
+
+    "envoy.grpc_credentials.access_token_example":      "//source/extensions/grpc_credentials/example:config",
```

Comment:
I think we should not build this one in by default, since most folks won't use it. It should only be included as a direct dependency for tests or the like.